account_invoice_ubl/models/ir_actions_report.py

```diff
@@ -29,13 +29,8 @@ def _post_pdf(self, save_in_attachment, pdf_content=None, res_ids=None):
         return pdf_content
 
     def render_qweb_pdf(self, res_ids=None, data=None):
```

Comment:
With this change, you need this function?

Reply:
It is an standard function, used to print any qweb report

Reply:
Actually, you are right, we can get rid of all the function here.